### ISSUE TITLE
windows spleeter-native stuck in 'running'

### ISSUE THREAD

**Issue by diejung** [diracdeltas/spleeter4max#32]
After a LOT of tinkering, I was able to actually get everything installed correctly (or so I thought) and I'm still having issues with getting this up and running. I'll run it in ableton and it'll do the "Spleeter is running, this may take a minute" with no luck. I've tried replacing the spleeter separate function with Python -m spleeter separate, no luck. I have both spleeter and the files im trying to split on the same drive, my python and ffmpeg are on my OS drive, no dice. Tried deleting the execSync variable, no luck. I tried installing the checker (where my spleeter is installed) and it gives me this below: 

![image](https://user-images.githubusercontent.com/66368814/83692508-c45a3380-a5b9-11ea-8a31-0979ecc9932e.png)

Here are some more screenshots showing where everything is set up, including my file directory path, CMD prompt verifications, etc. 

![image](https://user-images.githubusercontent.com/66368814/83692645-04211b00-a5ba-11ea-979f-40010cc675d5.png)

![image](https://user-images.githubusercontent.com/66368814/83692681-169b5480-a5ba-11ea-9e25-8c56f742fc55.png)

![image](https://user-images.githubusercontent.com/66368814/83692716-261a9d80-a5ba-11ea-9934-5b98de994ef4.png)

![image](https://user-images.githubusercontent.com/66368814/83692772-3d598b00-a5ba-11ea-8f2c-9f895559aaf0.png)

![image](https://user-images.githubusercontent.com/66368814/83692831-5bbf8680-a5ba-11ea-92c0-f780c632ad22.png)

I'm kind of at a loss as to where to go from here, as I've spent the past day or so trying to get this to work and doing self-troubleshooting with no luck getting past this point. Any help would be greatly appreciated.

**Max Console Screenshots**

![image](https://user-images.githubusercontent.com/66368814/83692286-4f86f980-a5b9-11ea-997b-98b381c2abdf.png)

**Info:**
 - Using spleeter-native
 - OS: Windows 10 Home
 - Ableton version 10.14
 - Max version 8.1.4

**Comment by diracdeltas** [comment]
@diejung 

1. 'no function set' is a benign error message and expected; it shouldn't cause any issues
2. in your first screenshot, you shouldn't be running pip install. instead download the check-install.py script to your downloads folder, then open CMD.exe, change into the downloads folder (using `cd` command), then run `python check-install.py`
3. what does `spleeter -h` show in CMD.exe?

if you need help with any of this feel free to ask in the spleeeter max channel in discord: https://discord.gg/x5YjA9

**Comment by diracdeltas** [comment]
also if you copy the command in max console that starts with `spleeter separate -i...`, paste it in CMD.exe, then hit enter, what happens?

**Comment by diejung** [comment]
Sorry for the late reply, this is what happens when I run my check install after actually installing the script, I'm not terribly knowledgeable on github or coding / scripting in general. 

![image](https://user-images.githubusercontent.com/66368814/83705287-2b87e000-a5da-11ea-8edf-079f3adbf146.png)

this is spleeter -h: 

![image](https://user-images.githubusercontent.com/66368814/83705310-3773a200-a5da-11ea-9b13-e241f6625df5.png)

this is with spleeter separate -i no variables changed through just cmd as admin -

![image](https://user-images.githubusercontent.com/66368814/83705409-7144a880-a5da-11ea-8135-be57c8c77ee9.png)
 
this is what happens when i run cmd in the install folder and do the same command - 

![image](https://user-images.githubusercontent.com/66368814/83705429-86213c00-a5da-11ea-9470-a94d7ba6885a.png)

I tried adding in what i thought would be the correct command for a random reference track and this is what it spat out: 

![image](https://user-images.githubusercontent.com/66368814/83705528-d4ced600-a5da-11ea-885b-f0a804775c9b.png)

**Comment by diracdeltas** [comment]
@diejung your last command is almost correct, just replace the ${} symbols. should look like this:

<img width="268" alt="Screen Shot 2020-06-03 at 8 04 13 PM" src="https://user-images.githubusercontent.com/549654/83710372-6e938480-a5d5-11ea-9681-0668ab03d53c.png">

**Comment by diejung** [comment]
Took a little bit of time to try and get the VST up and running and that was even more confusing, I ran it and it seems it correctly sent the stems to the spleeter-native folder i have installed.

![image](https://user-images.githubusercontent.com/66368814/83711283-64c74c80-a5e8-11ea-9dc5-e6bc01a30671.png)

**Comment by diejung** [comment]
I am in the discord btw if that makes troubleshooting easier! I'm die jung.

**Comment by diejung** [comment]
something new that comes up when I try to re-run it in Ableton:

![image](https://user-images.githubusercontent.com/66368814/83711942-38accb00-a5ea-11ea-9277-341fac977b2e.png)

**Comment by diracdeltas** [comment]
> something new that comes up when I try to re-run it in Ableton:

that's usually fixed by step 3 of https://github.com/diracdeltas/spleeter4max/tree/feature/native-spleeter#windows-steps.

if .JS is already removed from PATHEXT, i'm not sure what could be wrong here.

**Comment by diejung** [comment]
js seemed to have reinstalled itself after trying to get the VST up and running, so I cleared that again so it no longer shows up in PATHEXT and rebooted. Tried again in Max with a different reference track and this is what came up: 

![image](https://user-images.githubusercontent.com/66368814/83713947-0a7dba00-a5ef-11ea-84df-d51fe545ee61.png)

Maybe I have to just go the cmd route? Was hoping that I could just make it easy via the m4l plugin, but maybe my computer doesn't want me to do that for whatever reason.
